src/main/java/TestFile.java

```diff
@@ -5,7 +5,7 @@ public String foo() {
 
     private static void incorrectAssignmentInIfCondition() {
         boolean value = false;
-        if (value = true) {
+        if (value = false) {
```

Comment:
[Checkstyle] WARNING: 'if' have incorrect indentation level 8, expected level should be 4.